api/grpc/pb/api.proto

```diff
@@ -20,8 +20,8 @@ syntax = "proto3";
 // generated code for grpc server and client.
 package pb;
 
-// Peer represents a peer in the off-chain network.
-message Peer {
+// Peer ID represents the data required to identify and communicate.
```

Comment:
```suggestion
// Peer ID represents the data required to identify and communicate
// with a participant in the the off-chain network.
```

The suggestion in this  comment contained two lines, I think one of the lines have been missed. Adding it again.